wordpress.org/public_html/wp-content/themes/pub/wporg-support/functions.php

```diff
@@ -39,6 +69,9 @@ function wporg_support_scripts() {
 	wp_enqueue_style( 'forum-wp4-style', get_stylesheet_uri(), [ 'dashicons' ], filemtime( __DIR__ . '/style.css' ) );
 	wp_style_add_data( 'forum-wp4-style', 'rtl', 'replace' );
 
+	wp_enqueue_style( 'wporg-parent-2021-style', 'https://developer.wordpress.org/wp-content/themes/wporg-parent-2021/build/style.css', [ 'wporg-global-fonts' ] );
+	wp_enqueue_style( 'wporg-parent-2021-block-styles', 'https://developer.wordpress.org/wp-content/themes/wporg-parent-2021/build/block-styles.css' );
```

Comment:
Need a better way of enqueueing these. This isn't a child theme, so it's not a dependency like in other themes.